app/experimenter/legacy-ui/core/css/experimenter.css

```diff
@@ -234,3 +234,21 @@ pre.json {
 .required-label.optional .text-danger {
   display: none;
 }
+
+.analysis-table {
+  font-size: 12pt;
+}
+
+.positive-significance {
+  color: green;
+  font-weight: bold;
+}
+
+.negative-significance {
+  color: red;
+  font-weight: bold;
+}
+
+.subscript {
+  font-size: 10pt;
+}
```

Comment:
I'm not sure if Bootstrap offers us anything outside of the box for this but, the less custom styles the more copy-and-pastable this'll be in `nimbus-ui` since I know Jody and I will make efforts to keep custom styles to an absolute minimum. No other comments here though if Bootstrap doesn't have some similar look besides what Jody's pointed out.